### PR TITLE
feat(react-components): Move m/ft button to the domain object card

### PR DESCRIPTION
**Pull request by nilscognite**
#### Type of change

Feat  

## Description :pencil:

See this conversation: https://cognitedata.slack.com/archives/G0115JQ8JM8/p1724831633376919

* I also did some refactoring in the `ObservationsDomainObject`, because the `Quantity `was missing
* Fix missing "Copy to clipboard" button.

## How has this been tested? :mag:

Open storybook/architecture
Make a crop box or a measurement.

### CODE REVIEW

File: react-components/src/components/Architecture/DomainObjectPanel.tsx [hunk 1]
```diff
@@ -37,16 +37,16 @@ export const DomainObjectPanel = (): ReactElement => {
...
+  }, [setCurrentDomainObjectInfo, commands]);
+
+  // Fore the getString to be updated
+  if (commands !== undefined && info !== undefined) {
     for (const command of commands) {
       if (command instanceof CopyToClipboardCommand)
         command.getString = () => toString(info, t, unitSystem);
     }
-  }, [setCurrentDomainObjectInfo, commands]);
+  }
```
Comment (haakonflatval-cognite) [inline react-components/src/components/Architecture/DomainObjectPanel.tsx:50]: What is this change for? I think the for-loop should still be inside the useEffect, as it seems like something that's unnecessary to do on every render. But Perhaps `unitSystem` should be in the dependency array?

**Review by haakonflatval-cognite — CHANGES_REQUESTED** [review]
Looks good, just a comment

File: react-components/src/components/Architecture/DomainObjectPanel.tsx (hunk 1 above)
Comment (nilscognite) [reply react-components/src/components/Architecture/DomainObjectPanel.tsx:50]: The promblem vas that the getString wasn't apply and the button disabled. I dont know why, but when I changed the code, it worked.

**Review by haakonflatval-cognite — APPROVED** [review]
LGTM :+1:

**Pull request merged**